onnx/defs/tensor/defs.cc

```diff
@@ -1085,7 +1085,24 @@ ONNX_OPERATOR_SET_SCHEMA(
             "T",
             OpSchema::all_tensor_types(),
             "Constrain input and output types to all tensor types.")
-        .SetDoc(Upsample_ver7_doc));
+        .SetDoc(Upsample_ver7_doc)
+        .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
+          if (!hasNInputShapes(ctx, 1)) {
+            return;
+          }
+          propagateShapeAndTypeFromFirstInput(ctx);
```

Comment:
You probably only want to copy the input type to output here?

Reply:
My original intention is to prepare an output object for modifying. I will use propagateElemTypeFromInputToOutput() for less code instructions.
Thanks for your advice.